user-docs/latest/users-roles-permissions/configuring-administrator-roles.md

```diff
@@ -0,0 +1,160 @@
+# Configuring administrator roles
+
+Administrators are the users of an admin panel of a Strapi application. Administrator accounts and roles are managed with the Role-Based Access Control (RBAC) feature. It is available in the *Administration panel* category of the Settings interface, accessible from *General > Settings* in the main navigation of the admin panel.
+
+The *Administration panel* category is divided into 2 sub-categories: *Roles* and *Users* (see [Managing administrators](managing-administrators.md)).
+
+![Administrator roles interface](../assets/users-permissions/administrator_roles.png)
+
+The *Roles* sub-category of *Administration panel* displays all created roles for the administrators of your Strapi application.
+
+From this interface, it is possible to:
+
+- create a new administrator role (see [Creating a new role](#creating-a-new-role)),
+- delete an administrator role (see [Deleting a role](#deleting-a-role)),
+- or access information regarding an administrator role, and edit it (see [Editing a role](#editing-a-role)).
+
+By default, 3 administrator roles are defined for any Strapi application:
+
+- Author: to be able to create, manage and publish their own content.
+- Editor: to be able to create content, and manage and publish any content.
+- Super Admin: to be able to access all features and settings. This is the role attributed by default to the first administrator at the creation of the Strapi application.
+
+::: warning ATTENTION
+If you use your Strapi application with the Community Edition (see [Pricing and Plans](https://strapi.io/pricing)), your use of the RBAC feature will be limited. Only the 3 default roles are indeed available, as you cannot create more roles and cannot delete the default ones. It is however possible to edit them, but to an extent:
+
+- Configuring permissions in detail is only available for the Enterprise Edition: with the Community Edition, if you have access to a content type, it is automatically a full access with all permissions.
```

Comment:
Here, do we have to mention the `all permissions` at the "field level"
Meaning that it's full permissions (create, read, update and delete) but you can select fields you have access too.
I don't know if it's worth it to go at this detail of information in that warning box.

Reply:
Yes I think it's important to be very clear on the limitations of the Community Edition. I've changed the sentence to make it clearer.